File: minion/app/src/main/java/snakesoft/minion/Activities/MainActivity.java
package snakesoft.minion.Activities;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import snakesoft.minion.Models.GlobalModel;
import snakesoft.minion.R;


public class MainActivity extends AppCompatActivity implements View.OnClickListener
{

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mSyncBtn = (Button) findViewById(R.id.btnSync);
        mSyncBtn.setOnClickListener(MainActivity.this);

        mInboxBtn = (Button) findViewById(R.id.btnInbox);
        mInboxBtn.setOnClickListener(MainActivity.this);

        mActionsBtn = (Button) findViewById(R.id.btnActions);
        mActionsBtn.setOnClickListener(MainActivity.this);

        mSettingsBtn = (Button) findViewById(R.id.btnSettings);
        mSettingsBtn.setOnClickListener(MainActivity.this);

        mCalendarBtn = (Button) findViewById(R.id.btnCalendar);
        mCalendarBtn.setOnClickListener(MainActivity.this);

        mWpBtn = (Button) findViewById(R.id.btnWp);
        mWpBtn.setOnClickListener(MainActivity.this);

        GlobalModel.init(getApplicationContext());
    }

    @Override
    public void onClick(View v)
    {
        if(v == mInboxBtn)
        {
            Intent myIntent = new Intent(MainActivity.this, InboxActivity.class);
            MainActivity.this.startActivity(myIntent);
        }
        if(v == mSyncBtn)
        {
            Intent myIntent = new Intent(MainActivity.this, SyncActivity.class);
            MainActivity.this.startActivity(myIntent);
        }
        if(v == mActionsBtn)
        {
            Intent myIntent = new Intent(MainActivity.this, ActionsActivity.class);
            myIntent.putExtra("Mode", "Normal");
            MainActivity.this.startActivity(myIntent);
        }
        if(v == mSettingsBtn)
        {
            Intent myIntent = new Intent(MainActivity.this, SettingsActivity.class);
            MainActivity.this.startActivity(myIntent);
        }
        if(v == mCalendarBtn)
        {
            Intent myIntent = new Intent(MainActivity.this, CalendarActivity.class);
            startActivity(myIntent);
        }
        if(v == mBacklogBtn)
        {
            Intent myIntent = new Intent(MainActivity.this, ActionsActivity.class);
            myIntent.putExtra("Mode", "Backlog");
            MainActivity.this.startActivity(myIntent);
        }
        if(v == mWpBtn)
        {
            Intent myIntent = new Intent(MainActivity.this, WpActivity.class);
            MainActivity.this.startActivity(myIntent);
        }
    }

    private Button mSyncBtn;
    private Button mInboxBtn;
    private Button mActionsBtn;
    private Button mSettingsBtn;
    private Button mCalendarBtn;
    private Button mBacklogBtn;
    private Button mWpBtn;
}

File: minion/app/src/main/java/snakesoft/minion/Activities/MainActivity.kt
package snakesoft.minion.Activities

import android.content.*
import android.graphics.Color
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.provider.Telephony
import android.util.Log
import android.widget.Toast
import kotlinx.coroutines.GlobalScope
import org.jetbrains.anko.AnkoViewDslMarker
import org.jetbrains.anko.backgroundColor
import org.jetbrains.anko.design.longSnackbar
import org.jetbrains.anko.setContentView
import org.jetbrains.anko.support.v4._DrawerLayout
import snakesoft.minion.ClipboardListenerService
import snakesoft.minion.Models.Event1

class MainActivity : AppCompatActivity()
{
    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)

        Intent(applicationContext, ClipboardListenerService::class.java)

        MainActivityUI().setContentView(this)
    }
}

class MainActivityUI : MateriaActivityUI<MainActivity>()
{
    override fun fillActivityUI(_DrawerLayout: @AnkoViewDslMarker _DrawerLayout, ctx: Context)
    {
        _DrawerLayout.backgroundColor = Color.CYAN
    }
}